source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -99,14 +110,54 @@ It is an error to specify more than one host name in a connection string with
 the ``mongodb+srv`` protocol, and the driver MUST raise a parse error and MUST
 NOT do DNS resolution or contact hosts.
 
-The driver MUST NOT attempt to connect to any hosts until the DNS query has
-returned its results.
+A driver MUST verify that in addition to the ``{hostname}``, the
+``{domainname}`` consists of at least two parts: the domain name, and a TLD.
+Drivers MUST raise an error and MUST NOT contact the DNS server to obtain SRV
+(or TXT records) if the full URI does not consist of at least three parts.
```

Comment:
This additional validation of the `domainname` is orthogonal to `srvMaxHosts`. We should create separate tickets for the validation and `srvMaxHosts` implementations.

Reply:
I agree that it is outside the scope of this PR, which is why I didn't want this change getting lost. We can create a separate DRIVERS ticket to track. You mentioned that it was "copied from the original text". Which original text are you talking about? I can create a DRIVERS ticket, but want to make sure we don't have one already if this text originated from another change.

Reply:
I believe this text goes back to the very first commits for this spec. See: https://github.com/mongodb/specifications/blame/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst#L84